### PR TITLE
fixes #8791 - pin i18n gem

### PR DESCRIPTION
**Pull request by mmoll**
This is also failing release_nightly_test_deb at the moment.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -24,6 +24,7 @@ gem 'secure_headers', '~> 1.3'
 gem 'safemode', '~> 1.2'
 gem 'fast_gettext', '~> 0.8'
 gem 'gettext_i18n_rails', '~> 1.0'
+gem 'i18n', '< 0.7.0'
```
Comment (dLobatog) [inline Gemfile]: @mmoll I think `~> 0.6, >= 0.6.4` is a bit better as suggested in Redmine

Comment (mmoll) [inline Gemfile]: with `gem 'i18n', '~> 0.6', '>= 0.6.4'` I'm getting 0.7.0, with `gem 'i18n', '< 0.7.0'` 0.6.11 as desired, no idea why (bundler 1.7.4 from the Debian/jessie package)

**Comment by mmoll** [comment]
`gem 'i18n', '~> 0.6.4'` is working as desired, thanks @eLobato

**Comment by dLobatog** [comment]
@mmoll Great, will merge if tests are :green_apple:

**Comment by dLobatog** [comment]
Merged as 40c46f1f8670e1ef834182fe53e670da8da42428, thanks @mmoll !

**Pull request closed without merging**